content/influxdb/v2/process-data/common-tasks/downsample/downsample-data-quix.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Downsample data with Quix Streams
+seotitle: Downsample data with Python and Quix Streams
+description: >
+  How to create Python service that downsamples data with Quix Streams.
+menu:
+  influxdb_v2:
+    name: Quix
+    parent: Downsample data
+    identifier: influxdb_v2-downsample-quix
+weight: 302
+influxdb/v2/tags: [tasks]
+---
+
+A common practice when processing high volume data is to downsample it before comitting 
+it to InfluxDb to reduce the overall disk usage as data collects over time.
+
+This guide walks through the process of creating a series of Python services that ingest from InfluxDB v2, downsample and publish
+data to InfluxDB v3. By aggregating data within windows of time, then storing the aggregate values back to InfluxDB, you can reduce 
```

Comment:
If this doc is focused on downsampling data from v2 to v3, it should state that clearly in the title. If not, it should be structured as an alternative to Flux tasks for downsampling. The rest of this doc will likely need to be updated.

One side note, it might we worth creating and "Quix" section under "Tools & integrations" that introduces the basics of using Quix with InfluxDB v2, but then, as children, has docs for each of the specific workflows you can use.

Reply:
I will keep to v2 only. 
I'll do a Quix section in tools in a later PR.